Thrive.csproj

```diff
@@ -560,22 +563,19 @@
     <Folder Include="src\auto-evo\simulation\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json">
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1">
       <Version>13.0.1</Version>
     </PackageReference>
-    <PackageReference Include="System.Numerics.Vectors">
+    <PackageReference Include="System.Numerics.Vectors" Version="4.5.0">
       <Version>4.5.0</Version>
     </PackageReference>
-    <PackageReference Include="SharpZipLib">
+    <PackageReference Include="SharpZipLib" Version="1.3.3">
       <Version>1.3.3</Version>
     </PackageReference>
-    <PackageReference Include="Nito.Collections.Deque">
+    <PackageReference Include="Nito.Collections.Deque" Version="1.1.1">
       <Version>1.1.1</Version>
     </PackageReference>
-    <PackageReference Include="StyleCop.Analyzers">
-      <Version>1.2.0-beta.435</Version>
-    </PackageReference>
-    <PackageReference Include="Lib.Harmony">
+    <PackageReference Include="Lib.Harmony" Version="2.2.2">
```

Comment:
These changes should be reverted.

Reply:
I've tried a few times, but I revert them thrive won't load for me until it auto converts it back to this. Don't know how to fix that.

Reply:
Update everything starting from dotnet and msbuild, and make sure that Godot is configured to use the right tools.
Last time this kind of change attempted to happen it entirely broke the game for me for running through Godot. Maybe 4.0 can deal with this by using the dotnet 6 runtime but for now I don't think this change should be allowed.